### PR TITLE
feature: Seed Implementation

### PR DESCRIPTION
**Pull request by lolgab**

### CODE REVIEW

File: src/model/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// import { run } from "./codacyInput"
+
+// module.exports = {
+//   run,
+  
+// }
```
Comment (josemiguelmelo) [inline src/model/index.ts:6]: You can remove this file

File: src/test/index.spec.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// import { Codacyrc } from "../model/codacyInput";
+// import { Engine } from "../engine";
+
+// const mockEngine: Engine = async function(codacyrc: Codacyrc) {
+
+// }
```
Comment (josemiguelmelo) [inline src/test/index.spec.ts:6]: You can remove this file

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,24 @@
+ARG NODE_IMAGE_VERSION=13-alpine
```
Comment (franciscodua) [inline Dockerfile:1]: Why? We'll always update the Dockerfile to bump the version, right?

File: src/model/codacyInput.ts [hunk 4]
```diff
@@ -0,0 +1,41 @@
+export type ParameterValue = any
+
+export class Parameter {
```
Comment (franciscodua) [inline src/model/codacyInput.ts:3]: I was expecting a file per model class.
Why do we keep them all in the same file?

File: src/resultString.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { CodacyResult } from "./model/codacyResult"
+
+export function resultString(results: CodacyResult[]): string {
+  const lines = results.map(result => JSON.stringify(result))
+  return lines.join("\n")
+}
```
Comment (josemiguelmelo) [inline src/resultString.ts:6]: You could add a `toString` method to CodacyResult model instead

File: Dockerfile (hunk 3 above)
Comment (lolgab) [reply Dockerfile]: This was because of copy pasting. Removed ✅

File: src/model/codacyInput.ts (hunk 4 above)
Comment (lolgab) [reply src/model/codacyInput.ts:3]: I grouped classes by "topic" to not have too many files.

File: src/model/index.ts (hunk 1 above)
Comment (lolgab) [reply src/model/index.ts]: Done ✅

File: src/test/index.spec.ts (hunk 2 above)
Comment (lolgab) [reply src/test/index.spec.ts]: Done ✅

File: src/resultString.ts (hunk 5 above)
Comment (lolgab) [reply src/resultString.ts:6]: It takes an array of results and not a single result.

Comment (josemiguelmelo) [reply src/resultString.ts:6]: You're right 👍

**Pull request merged**